src/Features/Core/Portable/QuickInfo/CommonSemanticQuickInfoProvider.cs

```diff
@@ -365,10 +378,10 @@ private ImmutableArray<ISymbol> GetSymbolsFromToken(SyntaxToken token, Workspace
                 .GetSymbols(includeType: true);
         }
 
-        private static bool IsOk(ISymbol symbol)
+        private static bool IsOk([NotNullWhen(returnValue: true)] ISymbol? symbol)
             => symbol != null && !symbol.IsErrorType();
 
-        private static bool IsAccessible(ISymbol symbol, INamedTypeSymbol within)
+        private static bool IsAccessible(ISymbol symbol, INamedTypeSymbol? within)
```

Comment:
```suggestion
        private static bool IsAccessible(ISymbol symbol, [NotNullWhen(false)] INamedTypeSymbol? within)
```

Reply:
So this one isn't wrong, but I'll admit the annotation seems odd and unlikely to be useful. I'm kinda willing to leave this one off and honestly wish we redid the signature.

Reply:
I agree with Jason on this. The annotation seems like it would result in "magic" down the road.